doc/content/source/bcs/SecondaryElectronEnergyBC.md

```diff
@@ -1,20 +1,41 @@
 # SecondaryElectronEnergyBC
 
-!alert construction title=Undocumented Class
-The SecondaryElectronEnergyBC has not been documented. The content listed below should be used as a starting point for
-documenting the class, which includes the typical automatic documentation associated with a
-MooseObject; however, what is contained is ultimately determined by what is necessary to make the
-documentation clear for users.
-
 !syntax description /BCs/SecondaryElectronEnergyBC
 
 ## Overview
 
-!! Replace these lines with information regarding the SecondaryElectronEnergyBC object.
+`SecondaryElectronEnergyBC` is an electron mean energy density of secondary electrons induced by ion flux outflow boundary condition.
+`SecondaryElectronEnergyBC` assumes the electrostatic approximation for the electric field.
+
+\begin{equation}
+a =
+\begin{cases}
+1, & \mu_{e} \left( \nabla (V) \right) \cdot \textbf{n} > 0\\
+0, & \mu_{j} \left( \text{-} \nabla (V) \right) \cdot \textbf{n} \leq 0\\
+\end{cases} \\[10pt]
```

Comment:
```suggestion
\begin{cases}
1, & \mu_{e} \left( -  \nabla (V) \right) \cdot \textbf{n} > 0\\
0, & \mu_{j} \left( \text{-} \nabla (V) \right) \cdot \textbf{n} \leq 0\\
\end{cases} \\[10pt]
```

I wasn't sure if leaving it off was intentional or not. If it is intentional just ignore this.